pkg/madmin/API.md

```diff
@@ -62,8 +62,25 @@ __Parameters__
 |`secretAccessKey`  | _string_  |Secret key for the object storage endpoint.   |
 |`ssl`   | _bool_  | Set this value to 'true' to enable secure (HTTPS) access.  |
 
+## 2. Admin API Version
 
```

Comment:
Get rid of APIs from docs which are not available anymore like HealUpload() ?

Reply:
Fixed the table - that was the place where old funcs were still present.